src/main/java/org/embulk/output/s3_per_record/S3PerRecordOutputPlugin.java

```diff
@@ -129,6 +136,7 @@ public S3PerRecordPageOutput(PluginTask task, Schema schema) {
                 credentials = new DefaultAWSCredentialsProviderChain().getCredentials();
             }
             transferManager = new TransferManager(credentials);
+            logger.info(String.format("Start Upload to bucket \"%s\"", bucket));
```

Comment:
`{}` interpolation will be better.

```java
logger.info(String.format("Start Upload to bucket \"{}\"", bucket));
```

http://www.slf4j.org/manual.html#typical_usage